contributors/guide/contributor-cheatsheet/README-ja.md

```diff
@@ -290,6 +290,15 @@ git checkout -b myfeature
 [スカッシュコミット]の主な目的はきれいに読めるgitの履歴や加えられた変更のログを作成することです。通常これはPRの改定の最終段階に行われます。
 コミットをスカッシュするべきかわからない場合は、作業を止めてPRのレビューと承認を担当する他のコントリビューターの判断に任せることをおすすめします。
 
+`git rebase` を対話モードで実行して、保持するコミットとまとめるコミットを選択し、ブランチを強制的にプッシュします:
```

Comment:
Thank you for adding.
This sentence is not translated with 1 by 1 words, but It's OK for me and maybe most Japanese, I think.